terraform/per-project/variables.tf

```diff
@@ -50,6 +50,17 @@ variable "app_project_id" {
   default = ""
 }
 
+variable "terraform_dashboard" {
+  description = "If set to true, Terraform will create a Cloud Monitoring dashboard including important Spanner metrics."
+  type        = bool
+  default     = true
+}
+
+variable "dashboard_threshold_spanner_config" {
```

Comment:
Since the autoscaler allows for custom thresholds, perhaps the variable should just be the threshold itself and take a number instead. Doing so would allow the threshold to be more easily customized.